src/scss/_pulsar-date-picker.scss

```diff
@@ -0,0 +1,249 @@
+        }
+    }
+
+    .ui-datepicker-prev {
+        float: left;
+
+        &::before {
+            content: '\2190';
+            font-family: Arial;
+            font-size: 20px;
+            padding: 0 5px 5px 5px;
+        }
+    }
+
+    .ui-datepicker-next {
+        float: right;
+
+        &::after {
+            content: '\2192';
+            font-family: Arial;
+            font-size: 20px;
+            padding: 0 5px 5px 5px;
+        }
+    }
+
+    .ui-datepicker-title {
+        font-size: 14px;
+    }
+
+    .ui-datepicker-calendar {
+        border-collapse: collapse;
+        border-spacing: 0;
+        border: 0;
+        width: 100%;
+    }
+
+    td.ui-datepicker-today::after {
+        content: none;
+    }
+
+    .ui-datepicker-buttonpane {
+        margin-top: 10px;
+        text-align: right;
+    }
+
+    // Chosen date (normally based on value of date input)
+    .ui-state-active {
+        background-color: #3e3e3e;
+        color: #ffffff;
+    }
+
+    .ui-state-highlight {
+        background-color: #efefef;
+        color: #000000;
+    }
+
+    .ui-datepicker-close {
+        appearance: none;
+        background: transparent;
+        background-color: #dedede;
+        border: 0;
+        border-bottom-style: solid;
+        border-radius: 4px;
+        box-shadow: 0 2px 0 #c5c5c5;
+        -moz-box-sizing: border-box;
+        box-sizing: border-box;
+        color: #000000;
+        cursor: pointer;
+        display: inline-block;
+        font-size: 12px;
+        font-weight: normal;
+        line-height: normal;
+        margin: 0;
+        overflow: visible;
+        padding: 4px 5.6px 2.8px;
+        position: relative;
+        text-align: center;
+        text-decoration: none;
+        -moz-user-select: none;
+        -ms-user-select: none;
+        -webkit-user-select: none;
+        user-select: none;
+        vertical-align: top;
+        white-space: nowrap;
+
+        &::-moz-focus-inner {
+            border: 0;
+            padding: 0;
+        }
+
+        &:hover{
+            box-shadow: 0 1px 0 #c5c5c5;
+            transform: translateY(1px);
+
+            // Uses a pseudo-element to fill the gap that jitter on hover
+            &::after {
+                bottom: 0;
+                content: '';
+                height: calc(100% + 5px);
+                left: -.05%; // make it slightly wider to prevent side jitter
+                position: absolute;
+                top: -3px;
+                width: 101%; // make it slightly wider to prevent side jitter
+                z-index: -1;
+            }
+
+            &:focus {
+                box-shadow: 0 1px 0 #000000;
+            }
+
+            &:focus:active {
+                box-shadow: none;
+            }
+        }
+
+        &:focus {
+            background-color: #fd0;
+            box-shadow: 0 2px 0 #000000;
+            color: #000000;
+            outline: none;
+        }
+
+        &:active{
```

Comment:
space

Reply:
Do we not have an auto CS tool for sass? Same question for JS? Prevents us having to pick up these types of things in a PR review